Sources/Paywalls/Components/Common/Border.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  Border.swift
+//
+//  Created by Josh Holtz on 9/30/24.
+// swiftlint:disable missing_docs
+
+import Foundation
+
+#if PAYWALL_COMPONENTS
+
+public extension PaywallComponent {
+
+    struct Border: Codable, Sendable, Hashable {
```

Comment:
Just for my understanding: why is this an extension on `PaywallComponent`?

Reply:
Purely namespacing reasons 😇  If a developer does `import RevenueCatUI`, the `Border` would be directly available in the file and could cause some weird naming conflicts. So anything that is public and very generally named, we are trying to put under this namespace so its under `PaywallComponent.Border` 🤷‍♂️

Reply:
Oh right, namespacing. Makes sense, thanks!